<launch>

  <arg name="rviz" default="true" />

  <include file="$(find rslidar_pointcloud)/launch/rs16_seg.launch" />

  <node pkg="loam_velodyne" type="segmentMatching" name="segmentMatching" output="screen"/>
  <node pkg="loam_velodyne" type="odometryMaintenance" name="odometryMaintenance" output="screen"/>
  <node pkg="loam_velodyne" type="poseEstimateImu" name="poseEstimateImu" output="screen" respawn="true">

  </node>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find loam_velodyne)/rviz_cfg/loam_velodyne.rviz" />
  </group>

</launch>
